lte/cloud/go/services/nprobe/obsidian/models/swagger.v1.yml

```diff
@@ -292,3 +292,20 @@ definitions:
         type: string
         x-nullable: false
         example: '127.0.0.1:4040'
+
+  network_probe_data:
```

Comment:
Should this definition have any required attributes?

Reply:
fixed. could you help merge this pr please ?